source/common/dmtbdump2.c

```diff
@@ -588,6 +588,14 @@ AcpiDmDumpIvrs (
                     InfoTable = AcpiDmTableInfoIvrs8c;
                     break;
 
+                /* Variable-length entries */
+
+                case ACPI_IVRS_TYPE_HID:
+
+                    EntryLength = 22;
```

Comment:
regarding AMD I/O Virtualization Technology (IOMMU) Specification (48882—Rev 3.05-PUB—Jan 2020)
" EntryLength"  = Size of IVHD block in bytes, starting from Type field and including IVHD device entries.

By my understand "EntryLength" should be greater than 22. it should be  ( 40 + n +1), or I'm wrong ?

Reply:
22 is correct, see table 99 in the document. You are probably confusing entry length with the header that precedes a list of entries, which is described in table 89.

Reply:
indeed, 22 is correct.
Thank you.